doc/manual/src/command-ref/env-common.md

```diff
@@ -2,11 +2,11 @@
 
 Most Nix commands interpret the following environment variables:
 
-  - `IN_NIX_SHELL`\
+  - <a name="env-IN_NIX_SHELL" /> [`IN_NIX_SHELL`](#env-IN_NIX_SHELL)\
```

Comment:
Why does the anchor `env-IN_NIX_SHELL` need to be specified twice?

Reply:
That seems like an anti-pattern. We switched to Markdown to avoid verbose markup syntax like this. Actually this seems to be mixing content and presentation (having an easily copied link is something that should be done by the output generator, not something that the author needs to specify manually for every anchor).

Reply:
I can do that too if you prefer, I just chose this since the manual already uses this format:

https://github.com/NixOS/nix/blob/d8398d33c9a09e1f5599127ae6d477e7e0868b55/doc/manual/generate-builtins.nix#L9